lib/prism/parse_result.rb

```diff
@@ -86,7 +102,7 @@ def end_offset
 
     # The line number where this location starts.
     def start_line
-      source.line(start_offset)
+      source.line(start_offset) + 1
```

Comment:
It seems better that Source#line returns a 1-starting line.
I think it's a breaking change if Source#line returns a 0-starting line.

Reply:
`Location#start_line` and `Location#end_line` are the user-facing APIs and those are still 1-indexed, so nothing really changing that's user-facing here. This just avoids some extra computation internally. `Source#line` is not meant to be used on its own (which is why it's private on almost all of the APIs).